Dependency issue with development installation

Hi!

I've been trying to build the last version of Mamba on a fresh Ubuntu 20.04 VM, following the instructions in the readme. When running `pip install -e .`, I run into the following error:
```
    src/core/repo.cpp: In member function 'bool mamba::MRepo::read_file(const string&)':
    src/core/repo.cpp:247:60: error: 'CONDA_ADD_USE_ONLY_TAR_BZ2' was not declared in this scope
      247 |         int flags = Context::instance().use_only_tar_bz2 ? CONDA_ADD_USE_ONLY_TAR_BZ2 : 0;
          |                                                            ^~~~~~~~~~~~~~~~~~~~~~~~~~
    error: command 'gcc' failed with exit status 1
```

Somehow it seems related to 748060c. When running `mamba install ... libsolv ... -c conda-forge` from the readme, version 0.7.16 is installed on my machine but version >=0.7.18 is required since that commit.

--- a/README.md
+++ b/README.md
@@ -138,9 +138,9 @@ For more instructions (including OS X) check out https://gist.github.com/wolfv/f
 
 ### Development installation
 
-Make sure to have the following requirements in your conda environment:
+Make sure you install mamba's requirements in your conda environment:
 
-`mamba install cmake compilers pybind11 libsolv libarchive libcurl nlohmann_json pip cpp-filesystem yaml-cpp reproc-cpp -c conda-forge`
+`mamba env update --name <conda_env> --file environment-dev.yml`
 
 If you build mamba in a different environment than base, you must also install conda in
 that environment:

--- a/environment-dev.yml
+++ b/environment-dev.yml
@@ -7,7 +7,7 @@ dependencies:
   - ninja
   - nlohmann_json
   - pybind11
-  - libsolv
+  - libsolv >=0.7.18
   - libarchive
   - libcurl
   - cxx-compiler

